team-b-batch/src/main/kotlin/com/address/dsmproject/service/SaveAddressService.kt

```diff
@@ -0,0 +1,73 @@
+package com.address.dsmproject.service
+
+import com.address.dsmproject.feign.AddressClient
+import com.address.dsmproject.scheduler.AddressUtil.ENG_REAL_FILE_NAME
+import com.address.dsmproject.scheduler.AddressUtil.ENG_REQ_TYPE
+import com.address.dsmproject.scheduler.AddressUtil.ENG_ZIP_NAME
+import com.address.dsmproject.scheduler.AddressUtil.KOR_REAL_FILE_NAME
+import com.address.dsmproject.scheduler.AddressUtil.KOR_REQ_TYPE
+import com.address.dsmproject.scheduler.AddressUtil.KOR_ZIP_NAME
+import com.address.dsmproject.scheduler.AddressUtil.ZIP
+import org.springframework.stereotype.Service
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.time.LocalDateTime
+import java.util.zip.ZipFile
+
+@Service
+class SaveAddressService(
+    private val addressClient: AddressClient,
+) {
+
+    fun execute(reqType: String, zipFilePath: String, filePath: String) {
+        Files.write(Paths.get(zipFilePath), getAddressInfo(reqType))
+        val zipFile = File(zipFilePath)
+        val unzipFile = File(filePath)
+        unzipFile.mkdir()
+        unzip(zipFile, unzipFile)
+    }
+
+    private fun getAddressInfo(reqType: String): ByteArray {
```

Comment:
request param을 build하는 함수는 reqType 보다 language 와 year, month 를 받는게 추후 특정 달에 대해 배치를 수행하거나 다른 곳에서 재사용할 때에 유용할 것 같아요 + LocalDateTime.now() 라는 context 에서 분리 되어 조금 더 순수한 함수로 사용될 수 있을 것 같아요
```suggestion
    private fun getAddressInfo(language: String, year: Long, Month: Long): ByteArray {
```

Reply:
reqType보다 language라는 말씀은 네이밍에 관련한 comment일까요??